src/modules/UI/selectors.js

```diff
@@ -45,6 +45,26 @@ export const getArchivedWalletIds = (state: State): Array<string> => {
   return archivedWalletIds
 }
 
+export const getWalletLoadingPercent = (state: State) => {
+  const walletsForProgress = state.ui.wallets.walletLoadingProgress
```

Comment:
what are `walletsForProgress`?

Reply:
I believe I used that term in the case that we purposefully do not try to calculate the percentage for all of the wallets (eg archived wallets).